src/lib.rs

```diff
@@ -352,6 +353,13 @@ impl DecodeReaderBytesBuilder {
 /// buffer used to store the results of transcoding. Callers may elect to reuse
 /// the internal buffer via the `DecodeReaderBytesBuilder::build_with_buffer`
 /// constructor.
+///
+/// Because there is an internal buffer, the inner type `R` does not need to be
+/// buffered separately via `BufReader` - that is, you can wrap types such as
```

Comment:
Is this supposed to be an em-dash? I think I would just use a period here and start a new sentence with `That is, ...`.